uefi-sct/SctPkg/TestInfrastructure/SCT/Framework/ENTS/EasLib/EntsDebug.c

```diff
@@ -744,9 +744,8 @@ _LibMemoryMap (
   ASSERT (DescriptorVersion != NULL);
   //
   // Initialize for GrowBuffer loop
-  //
-  Buffer      = (EFI_MEMORY_DESCRIPTOR *)&Buffer;
   BufferSize  = sizeof (EFI_MEMORY_DESCRIPTOR);
+  Buffer      = __AllocatePool(BufferSize);

```

Comment:
I'm not sure why we need to call  __AllocatePool(). It looks like simply replacing it with "Buffer = NULL" should be good enough to fix the issue.